independent-publisher-connectors/Workable/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1692 @@
+              "type": "object",
+              "properties": {
+                "candidates": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "Candidate ID",
+                        "title": "Candidate ID"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "Name",
+                        "title": "Name"
+                      },
+                      "Firstname": {
+                        "type": "string",
+                        "description": "Firstname",
+                        "title": "Firstname"
+                      },
+                      "Lastname": {
+                        "type": "string",
+                        "description": "Lastname",
+                        "title": "Lastname"
+                      },
+                      "Headline": {
+                        "type": "string",
+                        "description": "Headline",
+                        "title": "Headline"
+                      },
+                      "account": {
+                        "type": "object",
+                        "properties": {
+                          "subdomain": {
+                            "type": "string",
+                            "description": "Subdomain",
+                            "title": "Subdomain"
+                          },
+                          "name": {
+                            "type": "string",
+                            "description": "Name",
+                            "title": "Name"
+                          }
+                        },
+                        "description": "account"
+                      },
+                      "job": {
+                        "type": "object",
+                        "properties": {
+                          "shortcode": {
+                            "type": "string",
+                            "description": "Short code",
+                            "title": "Short code"
+                          },
+                          "title": {
+                            "type": "string",
+                            "description": "Title",
+                            "title": "Title"
+                          }
+                        },
+                        "description": "job"
+                      },
+                      "stage": {
+                        "type": "string",
+                        "description": "Stage",
+                        "title": "Stage"
+                      },
+                      "disqualified": {
+                        "type": "boolean",
+                        "description": "Disqualified",
+                        "title": "Disqualified",
+                        "enum": [
```

Comment:
@DavidBreathe you have defined the enum type as "boolean"  but the values are defined as strings. please change to [true, false].

Reply:
Added 
"enum": [
                          "true",
                          "false"
                        ]

Reply:
@DavidBreathe You are adding the values as strings here please change to [true, false].

Reply:
updated